assets/js/theme/common/select-option-plugin.js

```diff
@@ -0,0 +1,50 @@
+import $ from 'jquery';
+
+/**
+ * Visually hides the option from user by moving option to an invisible
+ * and disabled select placeholder element.
+ *
+ * This approach is required rather than simply hiding the option because
+ * hidden option can still be included when serializeArray() is called and
+ * cause wrong value to be submitted.
+ * (eg. if you have option 1, 2, 3 and 2 is hidden, when you select 3,
+ * serializeArray() will use the value of 2 instead of 3)
+ */
+$.fn.toggleOption = function (show) {
```

Comment:
This is a copy of blueprint code that handles the hiding/showing of options in a dropdown